src/plans/details/tabs/Hooks/utils/utils.ts

```diff
@@ -169,3 +180,13 @@ export const validateHooks = (plan: V1beta1Plan): string => {
 
   return '';
 };
+
+export const getServiceAccountHelperText = (isPreHook: boolean, plan?: V1beta1Plan): string => {
+  const serviceAccountProject = isPreHook ? plan?.metadata?.namespace : plan?.spec?.targetNamespace;
+  const serviceAccountProjectLabel = isPreHook ? "plan's project." : "plan's target project.";
+
+  return t(
+    `Optional: Red Hat OpenShift service account. The service account must be provided if you want
```

Comment:
something in this sentence is confusing to me, we start with Optional and a few words after it must be provided. I think the word optional can removed, the red astrix near the field label should mean required, and without it, it should mean optional.
I would rephrase that to something like: Manipulate resources on the cluster by adding a service account. (or something like that) @jpuzz0 @wolfeallison